### ISSUE TITLE
Lists - Customization chips should be outlined

### ISSUE THREAD

**Issue by paulinea** [Orange-OpenSource/ods-android#248]

**Comment by paulinea** [comment]
Should be fixed in [#169](https://github.com/Orange-OpenSource/ods-android/issues/169)

**Comment by B3nz01d** [comment]
I do not see outlined chips in the list customization page
![Screenshot_20220823_180720_com.orange.ods.demo.jpg](https://user-images.githubusercontent.com/62606284/186207764-7df125da-f671-4918-9764-5b2acb1de3b6.jpg)

**Comment by paulinea** [comment]
@B3nz01d I move this issue cause as mentioned in a previous comment, this will be fixed in the #169

**Comment by paulinea** [comment]
We have to wait #169 merge before testing and moving it to Done column
